products/statement-generator/src/styles/utilityStyles.ts

```diff
@@ -47,7 +47,15 @@ const useUtilityStyles = makeStyles<Theme>(
         display: 'flex',
         flexDirection: 'row',
         width: '100%',
+        [breakpoints.down('sm')]: {
+          marginTop: 40,
+        },
+
+        [breakpoints.down('xs')]: {
+          marginTop: 40,
+        },
       },
+
```

Comment:
@davidwiese jk one last thing. Is this supposed to be here? I'm not seeing a button container on the Before you Being page, and we don't want unintended consequences on the rest of the buttons?

Reply:
Yea, that's adjusting the margin between the Before You Begin section and the Buttons on the bottom of the page for mobile layouts (part of the `<FlowNavigation />` component).

Reply:
@sydneywalcoff Ok I was just overthinking the problem, or was just misunderstanding the goal of the design change. 

My initial understanding was that the buttons should be a fixed distance from the alert, when really it's just the minimum distance that is desired.

Should be good to go now!